src/lidar_processor.cpp

```diff
@@ -76,37 +76,43 @@ void LidarProcessor::raw_pc_callback(const sensor_msgs::msg::PointCloud2::Shared
 
   // *** Begin filter ***
   // Container for original & filtered data
-  pcl::PointCloud<pcl::PointXYZ>::Ptr cloud(new pcl::PointCloud<pcl::PointXYZ>);
+  pcl::PointCloud<pcl::PointXYZI>::Ptr cloud(new pcl::PointCloud<pcl::PointXYZI>);
 
   // Convert to PCL data type
   pcl::fromROSMsg(*msg, *cloud);
 
   // Perform the Passthrough filtering
-  pcl::PassThrough<pcl::PointXYZ> pass;
+  pcl::PassThrough<pcl::PointXYZI> pass;
   pass.setInputCloud(cloud);
   pass.setFilterFieldName("z");
+  pass.setFilterLimits(0.0, 8.0);
+  pass.filter(*cloud);
+  
+  pass.setInputCloud(cloud);
+  pass.setFilterFieldName("intensity");
   pass.setFilterLimits(0.0, 0.5);
```

Comment:
is 0.0 to 0.5 gonna be good for intensity? this looks like its gonna filter out high intensity points

Reply:
Fixed:)